.github/workflows/functional-tests.yml

```diff
@@ -65,25 +65,25 @@ jobs:
 
       - name: Run cFS
         run: |
-          ./core-cpu1 | tee cFS_startup_cpu1.txt &
+          ./core-cpu1 &
           sleep 10
           ../host/cmdUtil --pktid=0x1806 --cmdcode=4 --endian=LE --string="20:CFE_TEST_APP" --string="20:CFE_TestMain" --string="64:cfe_testcase" --uint32=16384 --uint8=0 --uint8=0 --uint16=100 &
 
           sleep 30
-          counter=$(grep -c "CFE_TEST_APP" cFS_startup_cpu1.txt) &
+          counter=$(grep -c "BEGIN" cf/cfe_test.log) &
 
-          while [[ -z $(grep -i "SUMMARY" cFS_startup_cpu1.txt) ]]; do
+          while [[ -z $(grep -i "SUMMARY" cf/cfe_test.log) ]]; do
             echo "Waiting for CFE Tests"
             sleep 60
 
-            temp=$(grep -c "CFE_TEST_APP" cFS_startup_cpu1.txt) &
+            temp=$(grep -c "BEGIN" cf/cfe_test.log) &
               
             if [ $temp -eq $counter ]; then
               echo "Test is frozen. Quiting"
               break
             fi
 
-            counter=$(grep -c "CFE_TEST_APP" cFS_startup_cpu1.txt) &              
+            counter=$(grep -c "BEGIN" cf/cfe_test.log) &              
           done
```

Comment:
The file existence indicates complete, no loop required just loop on existence check up to X times.

Reply:
fixed